cmd/scollector/collectors/processes_windows.go

```diff
@@ -40,7 +40,7 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 	}
 
 	var svc_dst []Win32_Service
-	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	var svc_q = wmi.CreateQuery(&svc_dst, "")
```

Comment:
See https://msdn.microsoft.com/en-us/library/aa394606%28v=vs.85%29.aspx for more about WQL.